creature/badooga; Better Greatwyrms.json

```diff
@@ -0,0 +1,2137 @@
+					]
+				},
+				{
+					"name": "Frightful Presence",
+					"entries": [
+						"Each creature of the greatwyrm's choice that is within 1 mile of the dragon and aware of it must succeed on a {@dc 19} Wisdom saving throw or become {@condition frightened} for 1 minute. A creature can repeat the saving throw at the end of each of its turns, ending the effect on itself on a success. If a creature's saving throw is successful or the effect ends for it, the creature is immune to the greatwyrm's Frightful Presence for the next 24 hours."
+					]
+				},
+				{
+					"name": "Frost Breath {@recharge 5}",
+					"entries": [
+						"The greatwyrm exhales an icy blast in a 300-foot cone. Each creature and object in that area takes 88 ({@damage 16d10}) cold damage ({@dc 25} Constitution save for half). Damaged surfaces in this area then become {@hazard slippery ice} (DMG 110) until cleared."
+					]
+				}
+			],
+			"reaction": [
+				{
+					"name": "Vicious Reprisal",
+					"entries": [
+						"In response to taking damage, the greatwyrm makes a bite attack against a random creature within 5 feet of it. If no creature is within reach, the greatwyrm can attack an object, or it can move up to half its speed toward an enemy it can see, without provoking opportunity attacks."
+					]
+				}
+			],
+			"legendary": [
+				{
+					"name": "Tail Attack",
+					"entries": [
+						"The greatwyrm makes two tail attacks, one of which must be against an object."
+					]
+				},
+				{
+					"name": "Savage (Costs 2 Actions)",
+					"entries": [
+						"The greatwyrm makes a claw attack against each creature and object within 10 feet of it."
+					]
+				},
+				{
+					"name": "Wing Attack (Costs 2 Actions)",
+					"entries": [
+						"The greatwyrm beats its wings. Each creature within 10 feet of the greatwyrm must succeed on a {@dc 26} Dexterity saving throw or take 17 ({@damage 2d6 + 10}) bludgeoning damage and be knocked {@condition prone}. The greatwyrm can then fly up to half its flying speed."
+					]
+				}
+			],
+			"legendaryGroup": {
+				"name": "White Dragon",
+				"source": "MM"
+			},
+			"fluff": {
+				"_monsterFluff": {
+					"name": "Chromatic Greatwyrms",
+					"source": "BeGr"
+				},
+				"images": [
+					{
+						"type": "image",
+						"href": {
+							"type": "external",
+							"url": "img/bestiary/MM/White%20Dragon.jpg"
+						}
+					}
+				]
+			},
+			"dragonCastingColor": "W",
+			"tokenUrl": "img/MM/Adult White Dragon.png",
+			"senseTags": [
+				"U"
+			],
+			"actionTags": [
+				"Breath Weapon",
+				"Frightful Presence",
+				"Multiattack",
+				"Shapechanger"
+			],
+			"languageTags": [
+				"C",
+				"DR"
+			],
+			"damageTags": [
+				"P",
+				"C",
+				"S",
+				"B"
+			],
+			"miscTags": [
+				"MW",
+				"RCH",
+				"AOE"
+			],
+			"conditionInflict": [
+				"frightened",
+				"grappled",
+				"prone",
+				"restrained"
+			],
+			"conditionInflicted": [
```

Comment:
Should be `conditionInflict`
Issue repeated a few times throughout the file

Reply:
You mean, I should remove those entries from all the stat blocks? I believe they came from the Homebrew Builder

Reply:
So that's where all those have been coming from

Reply:
Watch out, normally the last instance of this issue in the file is not duplicated with the correct name and just has the wrong name.